requirements.txt

```diff
@@ -1,21 +1,21 @@
-wheel==0.36.2
-numpy==1.19.5
-pandas==1.2.0
-xgboost==1.3.2
-lightgbm==3.1.1
-catboost==0.24.4
-hyperopt==0.2.5
-scikit-learn==0.24.1
+wheel>=0.36.2
+numpy>=1.19.5
+pandas>=1.2.0
+xgboost>=1.3.2
+lightgbm>=3.1.1
+catboost>=0.24.4
+hyperopt>=0.2.5
+scikit-learn>=0.24.1
 matplotlib==3.3.3
 pdpbox==0.2.0
-kaleido==0.1.0
+kaleido>=0.1.0
 requests==2.25.1
 requests_cache==0.5.2
 plotly==4.14.3
 seaborn==0.11.1
-shap==0.37.0
+shap>=0.37.0
```

Comment:
Did you try to use insolver with shap == 0.38.1?

Our CI show an error when running test with latest shap package:
```
insolver/wrappers/gbm.py:10: in <module>
    from shap import TreeExplainer, summary_plot
/opt/hostedtoolcache/Python/3.8.6/x64/lib/python3.8/site-packages/shap/__init__.py:38: in <module>
    from .plots._beeswarm import summary_legacy as summary_plot
/opt/hostedtoolcache/Python/3.8.6/x64/lib/python3.8/site-packages/shap/plots/__init__.py:8: in <module>
    from ._image import image, image_to_text
/opt/hostedtoolcache/Python/3.8.6/x64/lib/python3.8/site-packages/shap/plots/_image.py:15: in <module>
    from IPython.core.display import display, HTML
E   ModuleNotFoundError: No module named 'IPython'
```
But no errors with shap==0.37.0

Reply:
Thanks!
```suggestion
shap==0.37.0
```